src/kili/utils/pagination.py

```diff
@@ -186,10 +186,13 @@ def generate_variables(batched_properties):
     batch = None
     for batch_number, batch in enumerate(batch_object_builder(properties_to_batch, batch_size)):
         payload = generate_variables(batch)
-        result = api_throttle(self.auth.client.execute)(request, payload)
+        try:
+            result = api_throttle(self.auth.client.execute)(request, payload)
+        except (TransportQueryError, GraphQLError) as err:
+            raise ValueError(
```

Comment:
Why don't we rethrow a TransportQueryError here?
it is one that we send when we don't execute mutations by batch isn't it?

Reply:
self.auth.client.execute could fail for GraphQLError if the graphql query is invalid according to the schema

or can fail for TransportQueryError if the query is refused by the backend

Reply:
I raise a value error but the `from err` will also add the original error (TransportQueryError or GraphQLError)